src/plugins/workspace/public/components/utils/workspace.ts

```diff
@@ -23,6 +23,12 @@ export const navigateToWorkspaceDetail = (
   );
 };
 
+export const navigateToWorkspaceListWithUseCase = ({ application }: Core, useCaseTitle: string) => {
```

Comment:
I found this function hard to use in my local because it requires caller to pass in `http`, while the http is not used in the function actually.

Reply:
Oh I missed it, updated.